### PR TITLE
Added classes and made code changes to allow merging various metrics …

### PR DESCRIPTION
**Pull request by skashin**
…files

### CODE REVIEW

File: src/tests/java/org/broadinstitute/dropseqrna/utils/TestUtils.java [hunk 1]
```diff
@@ -98,4 +103,42 @@ public static void assertSamFilesSame(final File actual, final File expected) {
...
+        File outputBAMList = getTempReportFile("SplitBamByCell", ".list");
+        bamSplitter.INPUT = Arrays.asList(inputBAM);
+        bamSplitter.OUTPUT = outputBAM;
+        bamSplitter.OUTPUT_LIST = outputBAMList;
+        bamSplitter.NUM_OUTPUTS = numOutputs;
+        Assert.assertEquals(bamSplitter.doWork(), 0);
+
+        List<File> splitBAMFileList = new ArrayList<>();
+        try {
+            List<String> filePathList = FileUtils.readLines(outputBAMList);
```
Comment (alecw) [inline src/tests/java/org/broadinstitute/dropseqrna/utils/TestUtils.java]: Can use IOUtil.readLines, which doesn't throw checked exception

**Comment by alecw** [comment]
Hi Seva,

Looks good.  Can you rebase and merge these commits to master?  Jim made pushed some changes earlier today that these need to be rebased onto.

Regards, Alec

**Comment by skashin** [comment]
Hi Alec,

  Sure, I will give it a try.

Best,
Seva

**Comment by skashin** [comment]
I rebased and merged the public repo code

**Pull request merged**